Frends.Kafka.Produce/Frends.Kafka.Produce/Definitions/Input.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.ComponentModel;
+
+namespace Frends.Kafka.Produce.Definitions;
+
+/// <summary>
+/// Input parameters.
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// Initial list of brokers as a CSV list of broker host or host:port. 
```

Comment:
Can we also include the "CSV list" in the example? Like

localhost:1234
OR
CSV_EXAMPLE